polyfill/lib/plaindatetime.mjs

```diff
@@ -32,15 +32,15 @@ export class PlainDateTime {
     nanosecond = 0,
     calendar = ES.GetISO8601Calendar()
   ) {
-    isoYear = ES.ToInteger(isoYear);
-    isoMonth = ES.ToInteger(isoMonth);
-    isoDay = ES.ToInteger(isoDay);
-    hour = ES.ToInteger(hour);
-    minute = ES.ToInteger(minute);
-    second = ES.ToInteger(second);
-    millisecond = ES.ToInteger(millisecond);
-    microsecond = ES.ToInteger(microsecond);
-    nanosecond = ES.ToInteger(nanosecond);
+    isoYear = ES.ToFiniteInteger(isoYear);
```

Comment:
Is “ToFiniteInteger” a thing in the spec text?

Reply:
It is currently not, my guess is that @Ms2ger is planning to add it as an editorial change as part of #1413 if it makes things more readable once the other spec text changes are complete?

Reply:
No concern about adding the AO, but it seems a bit strange to use it in the polyfill before there's spec text for it.

Reply:
Since it was pre-existing, I guess this would be appropriate for a follow-up.